lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -348,6 +348,14 @@ void IMConstPropPass::runOnOperation() {
       for (auto port : module.getBodyBlock()->getArguments())
         markOverdefined(port);
     }
+    // Also mark PlusArgs instrinsics as overdefined
+    for (auto &op : module.getOps()) {
+      if (isa<PlusArgsValueIntrinsicOp, PlusArgsTestIntrinsicOp>(op)) {
+        for (auto result : op.getResults()) {
+          markOverdefined(result);
+        }
```

Comment:
You can do the Scala-thing with:

```suggestion
        llvm::for_each(op.getResults(), [&](auto a) { markOverdefined(a); });
```